lib/queryUtil.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = {
+    count: function(domain, query, callback) {
+        var domainModel = require('../models/' + domain);
```

Comment:
Ahh, I figured there was a better way but I wasn't seeing it. Thanks, changed in fa5292e.